### ISSUE TITLE
WebPanel doesn't work on JFormDesigner

### ISSUE THREAD

**Issue by renato** [mgarin/weblaf#123]
After the upgrade to 1.27 with the new StyleManager, WebPanel (and propably the other components with the new manager, like WebLabel) stopped working inside JFormDesigner.

I tried to figure out why this happens inside JFormDesigner and not on my project itself but with no success.

Changing the WebPanel to JPanel temporarily solves the problem for some components that have no WebPanel specificity.

I understand you don't use JFormDesigner and I may investigate this issue further, I just would like to know if you have any direction to better understand this problem.

 ```
com.thoughtworks.xstream.mapper.CannotResolveClassException: com.alee.managers.style.data.SkinInfo
	at com.thoughtworks.xstream.mapper.DefaultMapper.realClass(DefaultMapper.java:56)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.DynamicProxyMapper.realClass(DynamicProxyMapper.java:55)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.PackageAliasingMapper.realClass(PackageAliasingMapper.java:88)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.ClassAliasingMapper.realClass(ClassAliasingMapper.java:79)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.ArrayMapper.realClass(ArrayMapper.java:74)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.MapperWrapper.realClass(MapperWrapper.java:30)
	at com.thoughtworks.xstream.mapper.CachingMapper.realClass(CachingMapper.java:45)
	at com.thoughtworks.xstream.core.util.HierarchicalStreams.readClassType(HierarchicalStreams.java:29)
	at com.thoughtworks.xstream.core.TreeUnmarshaller.start(TreeUnmarshaller.java:133)
	at com.thoughtworks.xstream.core.AbstractTreeMarshallingStrategy.unmarshal(AbstractTreeMarshallingStrategy.java:32)
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1058)
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1042)
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:988)
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:935)
	at com.alee.utils.XmlUtils.fromXML(XmlUtils.java:336)
	at com.alee.managers.style.skin.CustomSkin.<init>(CustomSkin.java:54)
	at com.alee.managers.style.skin.web.WebSkin.<init>(WebSkin.java:35)
	at com.alee.managers.style.StyleManager.getDefaultSkin(StyleManager.java:180)
	at com.alee.managers.style.StyleManager.getCurrentSkin(StyleManager.java:242)
	at com.alee.managers.style.StyleManager.applySkin(StyleManager.java:290)
	at com.alee.laf.panel.WebPanelUI.installUI(WebPanelUI.java:106)
	at javax.swing.JComponent.setUI(JComponent.java:664)
	at javax.swing.JPanel.setUI(JPanel.java:153)
	at com.alee.laf.panel.WebPanel.updateUI(WebPanel.java:906)
	at javax.swing.JPanel.<init>(JPanel.java:86)
	at javax.swing.JPanel.<init>(JPanel.java:95)
	at com.alee.laf.panel.WebPanel.<init>(WebPanel.java:164)
	at brax.desktop.MyCustomComponent.<init>(DSearchField.java:99)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at java.lang.Class.newInstance(Class.java:374)
	at com.jformdesigner.nn.a(SourceFile:68)
	at com.jformdesigner.op.k(SourceFile:177)
	at com.jformdesigner.op.b(SourceFile:185)
	at com.jformdesigner.sL.a(SourceFile:424)
	at com.jformdesigner.sL.a(SourceFile:475)
	at com.jformdesigner.sL.a(SourceFile:475)
	at com.jformdesigner.sL.b(SourceFile:253)
	at com.jformdesigner.qF.K(SourceFile:616)
	at com.jformdesigner.qF.a(SourceFile:385)
	at com.jformdesigner.qF.a(SourceFile:322)
	at com.jformdesigner.Mb.a(SourceFile:204)
	at com.jformdesigner.Mb.l(SourceFile:130)
	at com.jformdesigner.Mb.run(SourceFile:107)
```

**Comment by JFormDesigner** [comment]
This is a ClassLoader problem. IntelliJ IDEA uses different classloaders for plugins and for its core. And because IDEA includes XStream in its core, XStream is loaded from a different classloader than WebLaF and XStream can not find WebLaF classes, which causes the problems.

**Comment by JFormDesigner** [comment]
created PR #377 to fix this issue

**Comment by mgarin** [comment]
Thanks for clarifying the case!
I will merge your fix into `styling` branch slightly later today.

**Comment by mgarin** [comment]
I have merged PR #377 request so it is now available in `styling` branch.
New snapshot of this branch will be released today as well into the [snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/com/weblookandfeel/).

**Comment by samuelvisser** [comment]
@mgarin, was having the same problems and this fix solved it for me. But can you please release it to maven? The versions you have over there don't work and it would help me a lot to implement it the 'official' way.
